infrastructure/monitoring/README.md

```diff
@@ -1,312 +1,23 @@
-```
-3. Before applying your cluster configuration, you have to enter the correct value for the
-```yaml
-externalUrl: <>
-```
-```yaml
-storage:
-    volumeClaimTemplate:
-      metadata:
-        annotations:
-          name: prometheus-storage
-      spec:
-        resources:
-          requests:
-            storage: 50Gi
-        accessModes:
-          - ReadWriteOnce
-        storageClassName: rook-ceph-block
- ```
- ```yaml
-  affinity:
-    podAntiAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 99
-        podAffinityTerm:
-          labelSelector:
-            matchExpressions:
-            - key: app
-              operator: In
-              values:
-              - prometheus
-          topologyKey: kubernetes.io/hostname
-```
-
-### Monitor all namespaces
-We need to give permission to the pod prometheus to be able to get on endpoints that it does not know. To know them he has to talk with his APIserver and to do so he need an identity i.e. a ServiceAccount, finally the permissions are needed. To do this we use the concepts of ClusterRole and ClusterRoleBinding.
-So
-```bash
-$ kubectl apply -f prometheus-scraper-cluster-role.yaml
-```
-Now Prometheus can scrape all the namespaces in the cluster.
-
-
-### Blackbox monitoring
-Testing externally visible behavior as a user would see it.
-
-It allows us to tell if someone is seeing us or not and it is useful if something unexpected happens. We do the scraping from the outside in order to see that there are no access problems, for some reason, perhaps due to some crash.
-
-#### Blackbox exporter
-From [Blackbox exporter](https://github.com/prometheus/blackbox_exporter) official Github.
-The blackbox exporter allows blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP.
-TO deploy it, we use the [blackbox exporter chart](https://github.com/helm/charts/tree/master/stable/prometheus-blackbox-exporter) by appropriately changing the values of [values](https://github.com/helm/charts/blob/master/stable/prometheus-blackbox-exporter/values.yaml) file.
+Monitoring collects [Kubernetes manifests](https://kubernetes.io/docs/home/) and [Helm Charts](https://helm.sh/) to easy operate end-to-end Kubernetes cluster monitoring with [Grafana dashboard](https://grafana.com/), [Prometheus](https://prometheus.io/) using the Prometheus Operator, [Thanos](https://thanos.io/) and [Loki](https://grafana.com/oss/loki/) with [Promtail agent](https://grafana.com/docs/loki/latest/clients/promtail/).
 
-### Alert Notification
-When an alert is sent to AlertManager, those are also sent to Slack.
-This template differentiates alerts based on severity and sends them on the correct Slack channel.
+Monitoring is divided in two sub-directories:
+- [kube-prometheus-stack](./kube-prometheus-stack) is dedicated to collect  metrics. Metrics are a measurement for the system, they are crucial because they indicate the application usage and the platform's ability.
```

Comment:
```suggestion
- [kube-prometheus-stack](./kube-prometheus-stack) is dedicated to metrics collection. Metrics are a measurement for the system, they are crucial because they indicate the application usage and the platform's ability.
```